### PR TITLE
I18n settings

### PR DESCRIPTION
**Pull request by sbcgua**
related to #4415 

This field can be the list of texts to serialize and at the same time an indicator to use LXE stuff instead of old translations.

@mbtools @JustACasual 

2021-01-17_13h43_25

Prototype/proposal. But kind of solid one.

P.S. @mbtools adding settings is a charm now, thanks for your efforts :)

### CODE REVIEW

File: src/ui/zcl_abapgit_gui_page_sett_repo.clas.abap [hunk 1]
```diff
@@ -121,6 +122,10 @@ CLASS zcl_abapgit_gui_page_sett_repo IMPLEMENTATION.
       iv_label       = 'Main Language'
       iv_hint        = 'Main language of repository (cannot be changed)'
       iv_readonly    = abap_true
+    )->text(
+      iv_name        = c_id-i18n_langs
+      iv_label       = 'LXE langs (experimental)'
```
Comment (mbtools) [inline src/ui/zcl_abapgit_gui_page_sett_repo.clas.abap:127]: LXE > "Translations (experimental)"

"master language" > "main language"

I see you sneaked Ukranian in there 😀

Comment (sbcgua) [reply src/ui/zcl_abapgit_gui_page_sett_repo.clas.abap:127]: image
 What about this ?

Comment (mbtools) [reply src/ui/zcl_abapgit_gui_page_sett_repo.clas.abap]: Sure.

**Review by mbtools — APPROVED** [review]
👍

**Pull request merged**